centreon/src/Core/HostGroup/Infrastructure/Repository/DbWriteHostGroupActionLogRepository.php

```diff
@@ -227,6 +227,14 @@ public function unlinkFromHost(int $hostId, array $groupIds): void
         $this->writeHostGroupRepository->unlinkFromHost($hostId, $groupIds);
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function duplicate(int $hostGroupId, int $duplicateIndex): int
+    {
+        return $this->writeHostGroupRepository->duplicate($hostGroupId, $duplicateIndex);
```

Comment:
I'm not sure about the legacy behaviour, if you duplicate host groups should not we have action logs for each host groups created ?